src/services/CertificateGenerationService.ts

```diff
@@ -252,11 +316,31 @@ class CertificateGenerationService {
     }
 
     /**
-     * Retrieves the vehicle make and model for a given vehicle from the Technical Records microservice.
-     * Dramatically altered in CVSB-8582, in order to manage failure of vin data to conform to agreed formats
-     * @param testResult - the full test result record, from which vehicle attributes to search on can be obtained
+     * Method for getting make and model based on the vehicle from a test-result
+     * @param testResult - the testResult for which the tech record search is done for
      */
     public async getVehicleMakeAndModel(testResult: any) {
+        const techRecord = await this.getTechRecord(testResult);
```

Comment:
I know this wasn't really your change, but could this get the full coverage?
![image](https://user-images.githubusercontent.com/50110006/70246093-0f140380-176f-11ea-8693-e98d19b90e6b.png)